lib/xilinx/frames.cc

```diff
@@ -35,65 +38,75 @@ int Frames::readFrames(const std::string& frm_file_str) {
 
 		std::vector<std::string> frame_data_strings =
 		    absl::StrSplit(frame_delta.second, ',');
-		if (frame_data_strings.size() != 101) {
+
+		if (frame_data_strings.size() != ArchType::words_per_frame) {
 			std::cerr << "Frame " << std::hex << frame_address
 			          << ": found " << std::dec
 			          << frame_data_strings.size()
 			          << "words instead of 101";
```

Comment:
101 should be ArchType::words_per_frame?

Reply:
I am not quite sure what you mean and how you would like me to change this condition.
The ArchType::words_per_frame is an architecture specific setting and it's defined with an int in the architectures.h header file: https://github.com/SymbiFlow/prjxray/pull/1087/files#diff-f0d4cdb5eb101cb37373b1c37be1099dR34

Reply:
The print to std::cerr should not always print "101", it should print ArchType::words_per_frame, yes?

Reply:
Ohh, sure, wasn't looking at this part of the code snippet. Now it's perfectly clear what you meant.